src/components/_button.scss

```diff
@@ -8,10 +8,10 @@
   $type: 'primary',
   $size: 'medium',
   $layout: null,
-  $color: null,
-  $bgcolor: null,
-  $icon-fill: null,
-  $icon-stroke: null
+  $background-color: $c-gray-dark,
+  $color: $c-white,
+  $icon-fill: transparent,
+  $icon-stroke: $c-white
```

Comment:
In this case, I don't thing we need to use a name space because they are just params to define the `sui-button--custom` element.